persistence-modules/core-java-persistence-2/src/main/java/com/baeldung/insertnull/DBConfig.java

```
@@ -0,0 +1,30 @@
+package com.baeldung.insertnull;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class DBConfig {
+
+    private static Connection INSTANCE;
+
+    public static Connection getConnection() throws Exception {
```

Can we throw the proper Exception type here? SQLException